crates/ruff/src/rules/flake8_simplify/snapshots/ruff__rules__flake8_simplify__tests__SIM208_SIM208.py.snap

```diff
@@ -36,4 +36,64 @@ SIM208.py:4:4: SIM208 [*] Use `a == b` instead of `not (not a == b)`
 6 6 | 
 7 7 | if not a:  # OK
 
+SIM208.py:16:5: SIM208 [*] Use `b` instead of `not (not b)`
+   |
+16 |     pass
+17 | 
+18 | a = not not b  # SIM208
+   |     ^^^^^^^^^ SIM208
+19 | 
+20 | f(not not a)  # SIM208
+   |
+   = help: Replace with `b`
+
+ℹ Suggested fix
+13 13 | if not a != b:  # OK
+14 14 |     pass
+15 15 | 
+16    |-a = not not b  # SIM208
+   16 |+a = bool(b)  # SIM208
+17 17 | 
+18 18 | f(not not a)  # SIM208
+19 19 | 
+
+SIM208.py:18:3: SIM208 [*] Use `a` instead of `not (not a)`
+   |
+18 | a = not not b  # SIM208
+19 | 
+20 | f(not not a)  # SIM208
+   |   ^^^^^^^^^ SIM208
+21 | 
+22 | if 1 + (not (not a)):  # SIM208
+   |
+   = help: Replace with `a`
+
+ℹ Suggested fix
+15 15 | 
+16 16 | a = not not b  # SIM208
+17 17 | 
+18    |-f(not not a)  # SIM208
+   18 |+f(bool(a))  # SIM208
+19 19 | 
+20 20 | if 1 + (not (not a)):  # SIM208
+21 21 |     pass
+
+SIM208.py:20:9: SIM208 [*] Use `a` instead of `not (not a)`
+   |
+20 | f(not not a)  # SIM208
+21 | 
+22 | if 1 + (not (not a)):  # SIM208
+   |         ^^^^^^^^^^^ SIM208
+23 |     pass
+   |
+   = help: Replace with `a`
```

Comment:
The message and autofix title now no longer match what the fix is doing (I have a task to move the autofix title to `Fix`)